auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -270,7 +270,10 @@ def create_affiliation_invitation(
         if from_org_id == to_org_id:
             raise BusinessException(Error.DATA_ALREADY_EXISTS, None)
 
-        check_auth(org_id=from_org_id, one_of_roles=(ADMIN, COORDINATOR, STAFF))
+        if affiliation_invitation_type == AffiliationInvitationType.EMAIL:
+            check_auth(org_id=from_org_id, one_of_roles=(ADMIN, COORDINATOR, USER, STAFF))
+        if affiliation_invitation_type == AffiliationInvitationType.REQUEST:
```

Comment:
re-usable function to do this since we are doing the same thing in a few places here?